logsearchapi/server/partitions.go

```diff
@@ -242,3 +242,23 @@ func (c *DBClient) vacuumData(diskCapacityGBs int) {
 
 	}
 }
+
+func (c *DBClient) partitionTables() {
+	checkInterval := 12 * time.Hour
+	ticker := time.NewTicker(checkInterval)
+	bgCtx := context.Background()
+	for when := range ticker.C {
+		halfDayLater := when.Add(12 * time.Hour)
+		for _, table := range []Table{auditLogEventsTable, requestInfoTable} {
+			partitionExists, err := c.checkPartitionTableExists(bgCtx, table.Name, halfDayLater)
+			log.Printf("Error while checking if partition for %s exists %s", table.Name, err)
```

Comment:
For logging since is is part of k8s start using klog? which provides more k8s friendly logging?